subt/docker/unittest/run_solution.bash

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+. /opt/ros/melodic/setup.bash
+. ~/subt_solution/install/setup.sh
+
+# Wait for the bridge
+sleep 30
```

Comment:
This code is based on original example:
https://bitbucket.org/osrf/subt_seed/src/default/docker/subt_solution/run_solution.bash

If `--wait` does what we need I would definitely try it and also recommend it to the Alfredo ...

Reply:
Well the [documentation says](http://wiki.ros.org/roslaunch/Commandline%20Tools#line-45) it will wait for `roscore` which [might be the same thing](http://wiki.ros.org/roscore).

Reply:
It looks like that it at least works locally
```
developer@gpu-desktop:~/subt_solution$ ./osgar/subt/docker/unittest/run_solution.bash
roscore/master is not yet running, will wait for it to start
```
I and I surely prefer proper initialization and not sleep, thanks :+1: